Resources/Textures/Mobs/Species/Vox/organs.rsi/meta.json

```diff
@@ -20,6 +20,9 @@
         },
         {
             "name": "lung-r"
-        }
+        },
+		{
+			"name": "brain"
```

Comment:
We don’t use tabs, so you’ll need to replace these with spaces to match with the earlier alignment.